tests/sympc/tensor/mpc_tensor_test.py

```diff
@@ -23,6 +23,29 @@ def test_mpc_tensor_exception(get_clients) -> None:
         x = MPCTensor(secret=torch.Tensor([1, -2]), session=session)
 
 
+def test_mpc_tensor_nosession_exception(get_clients) -> None:
+    alice_client, bob_client = get_clients(2)
```

Comment:
MAYBE: Remove this

Reply:
Sorry for not making it clear. This was supposed to test for setup_mpc() not being called on the session.